zephyr/Kconfig

```diff
@@ -56,6 +56,19 @@ config ZEPHYR_DP_SCHEDULER
 	  DP modules can be located in dieffrent cores than LL pipeline modules, may have
 	  different tick (i.e. 300ms for speech reccognition, etc.)
 
+config ZEPHYR_TWB_SCHEDULER
+	bool "use Zephyr thread based TWB scheduler"
+	default n
+	depends on IPC_MAJOR_4
```

Comment:
why IPC4?
Why TGL is not able to use TWB?

Reply:
No reason for that, removed